llvm/cmake/modules/AddLLVM.cmake

```diff
@@ -2191,6 +2191,12 @@ function(llvm_install_library_symlink name dest type)
     set(LLVM_LINK_OR_COPY copy)
   endif()
 
+  if(LLVM_USE_SYMLINKS)
+    set(LLVM_LINK_OR_COPY create_symlink)
+  else()
+    set(LLVM_LINK_OR_COPY copy)
+  endif()
+
```

Comment:
This fragment seems the same as the one above. Probably the merge side effect. The commit is already present in the branch.

Reply:
fixed: I've deleted the commit